proposals/wac-ucr/index.bs

```diff
@@ -608,6 +843,14 @@ be immediately visible. Other posts should only be visible and editable to
 those who wrote them. They can then be viewable to the world when they get
 reviewed.
 
+<div class="assertion">
+
+* [[#req-inheritance-default-permissions]]
+* [[#req-agent-group]]
+
```

Comment:
"req-agent-group" is "The system shall allow access to be limited to an agent based on the agent’s membership in a certain group of agents."
That is true, but it is also true of everything, since we give access to groups of agents.

The key here is that we need a requirement on dynamic groups .
"The systems should allow permission to a resource to an agent that fits a description in a Linked Data Network"

Also another requirement for this to work is that the access control rule here be public so that the client can determine if it is in that larger network. 
This may involve sending a self signed credential that the user is in that network in order to help the Guard verify the claim (to avoid searching the whole network).